Makefile

```diff
@@ -147,7 +138,7 @@ generate-mocks:
 
 # this ensures there is no unused dependency being added by accident
 .PHONY: tidy
-tidy: crypto/relic/check
+tidy:
```

Comment:
Having the check in `tidy` make `make ci` fail since `install-tools` and `crypto/relic/check` are run in parallel. The crypto check fails to read the version from the relic directory because it wasn't created yet.

I suggest to leave `crypto/relic/check` for local development checks only. On CI. there is no risk to have mistakes since there is only one source of truth for the relic version.